lib/fastlane/plugin/datadog/actions/upload_symbols_to_datadog.rb

```diff
@@ -5,6 +5,7 @@ module Actions
     class UploadSymbolsToDatadogAction < Action
       def self.run(params)
         ENV['DATADOG_API_KEY'] = params[:api_key]
+        ENV['DATADOG_SITE'] = params[:site]
```

Comment:
i think `params[:site]` should be optional and we should change `ENV['DATADOG_SITE']` only if `params[:site] != nil`

Reply:
Agreed on making this optional 👍 
The parameter is already retrieved from the `ENV` [here](https://github.com/DataDog/datadog-fastlane-plugin/pull/4/files#diff-49d70083bf45b8944b258700389ac2d988ad129ad49aa0ad5002a0e34ecd012cR62) (same as the API key)

Reply:
i would do
```
if params[:site] is not nil {
  ENV['DATADOG_SITE'] = params[:site]
}
```
but maybe this is not needed, i don't know how it works.
what happens in this case?
```
export DATADOG_SITE=foo.com
fastlane run dd_lane(api_key:"key") // no "site" provided
```
does it upload to `foo.com`?
or does it upload to datadog-ci's default site?

Reply:
Yes, it will upload to `foo.com` 👍 
From [fastlane doc](https://docs.fastlane.tools/advanced/actions/):
> If the optional `env_name` is present, an environment variable with the specified name may also be used in place of an option in the Fastfile:
> 
> `MY_NEW_ACTION_FILE=file.txt fastlane run my_new_action)`

So we don't need to do `if params[:site] is not nil `